DIFFERENCES.md

```diff
@@ -50,3 +50,4 @@
   2021-10-14 09:38:39 <1> test-host(17235) [Ruby] connect/logger.rb:28
   Announcing system to http://smt-scc.nue.suse.com ...
   ```
+- HTTP debug logs are passed to default YaST logger not to STDERR.
```

Comment:
we should also doc somewhere that these 2 env vars are needed:
` Y2DEBUG=1 SCCDEBUG=1 yast registration`
and the debug output goes to `/var/log/YaST2/y2log`

Reply:
This works the same as before. No change there.